conf/messages.en

```diff
@@ -336,6 +336,8 @@ confirmation.paragraph.whatNext0.sample = We will give you your ruling within 30
 confirmation.paragraph.whatNext0.nosample = We will give you your ruling within 30 to 60 days.
 confirmation.paragraph.whatNext1 = You can sign in to check the status of your application.
 
+confirmation.refValue = GB{0}
```

Comment:
Why is this back?  GB prefix is no more and we are now using a common empty format for confirmation page and application pdf

Reply:
I have no idea what is this value.... Maybe the merge from master

Reply:
reference no tiene `GB`